src/screens/Data.js

```diff
@@ -245,7 +245,7 @@ const DataPage = () => (
           <PropertyValue type="boolean">
             <Description>
               Setting it to 'true' will include a Toolbar containing both
-              DataSearch and DataFilters with 'drop'.
+              DataSearch and DataFilters within a 'layer'.
```

Comment:
```suggestion
              DataSearch and DataFilters with `layer` prop.
```
Wondering how we might specify this is a prop applied